test/test-runner.js

```
@@ -0,0 +1,122 @@
+    if (!data.cf?.country || data.cf.country !== 'US') {
+      throw new Error('Invalid country in response');
+    }
+  }
+  
+  console.log('All tests passed!');
+}
+
+runTests().catch(console.error);
+
+// Test suite
+async function runTests() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'runTests'. Consider to delete it or rename it.

'runTests' is defined here:


(lint/suspicious/noRedeclare)